addons/udes_stock/models/stock_location.py

```diff
@@ -202,6 +202,28 @@ def _compute_storage_format(self):
     Computed field describing the state of the stock location.
     """,
     )
+    
+    u_size = fields.Float(
+        string="Location size",
+        help="""
+            The size of the location in square feet.
+        """,
+        digit=2,
+    )
+
+    u_size_write_date = fields.Datetime(
```

Comment:
Since this is not only about size changed, can we find a better name?

Reply:
Changed the name to be more specific.